### PR TITLE
add: p1-toast-components

### PR DESCRIPTION
**Pull request by gauravsinhaweb**
Add: p1-Toast

### CODE REVIEW

**Review by Mohitk391 — APPROVED** [review]
Nice UI, great work brother..

**Comment by gauravsinhaweb** [comment]
> Nice UI, great work brother..

Thank you :)

**Review by Mohitk391 — APPROVED** [review]
Just try to have more responsive design, everything else is great, can merge

File: index.html [hunk 1]
```diff
@@ -1312,8 +1312,173 @@ <h2 class="comp_head-text h3">List Category</h2>
...
+              type="text/plain"
+              class="line-numbers language-markup"
+              language-code="html"
+            >
+              <!-- snackbar - leading-->
+
+              <div class="snackbar_wrapper sb-leading">
+                <div class="sb-text">Your photo has been archived.</div>
+                <div><button class="sb-retry">undo</button></div>
+                <div><button class="sb-btn cross_leading">x</button></div>
```
Comment (mehulsatardekar) [inline index.html:1474]: Instead of adding Just **X** text if you add icons there, it will look better.

File: components/snackbar/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    0 1px 18px 0 rgb(0 0 0 / 12%);
+  border-radius: 4px;
+  color: var(--bg-color);
+  justify-content: space-around;
+  font-size: 1.4rem;
+  font-weight: 400;
+  align-items: center;
+}
+
+.sb-baseline {
```
Comment (mehulsatardekar) [inline components/snackbar/snackbar.css:31]: Here .sb-baseline and sb-leading are having the same property you can make a separate class for them.

**Comment by gauravsinhaweb** [comment]
> Just try to have more responsive design, everything else is great, can merge

Okay, I will do it.

**Review by mehulsatardekar — APPROVED** [review]
with the changes looks good to merge. BTW, UI is good and neat & I like the animation on navbar.

File: index.html (hunk 1 above)
Comment (gauravsinhaweb) [reply index.html:1474]: It's updated now. Thank you :)

File: components/snackbar/snackbar.css (hunk 2 above)
Comment (gauravsinhaweb) [reply components/snackbar/snackbar.css:31]: It's updated now.

**Comment by PuneetP16** [comment]
Nice UI great Work!!!

**Comment by gauravsinhaweb** [comment]
> Nice UI great Work!!!

Thank you! :)

**Pull request merged**